woocommerce/woocommerce-ios
# Enable Filters for product selector view

Closes: #8998 . Depends on https://github.com/woocommerce/woocommerce-ios/pull/9387 , which needs to be merged first

## Description
This PR enables filtering for product selection within Order creation. 

## Changes
We pass the existing `showFilters: true` property to the `ProductSelectorView` configuration. This will make avaialble the existing Filters implementation that is already been used for coupons.

## Screenshots

<img width=450 src="https://user-images.githubusercontent.com/3812076/231133623-e5ed82ff-fad4-4510-8bfe-dd0931ff1277.png">

## Testing instructions

- Go to Menu > Orders > `+` > `+ Add Products` >
- See that a new "Filter" button appears on the top right corner, and we can filter products in combination with search queries.

---
